docparse/output_walkthrough.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Output Walkthrough'
+description: 'Walking through a document using Aryn DocParse'
+icon: 'person-walking'
+---
+
+## Introduction
+
+In [this example](https://colab.research.google.com/drive/120STOQ5wGIpFUbgS7Gcld_PVW-CfLWOg#scrollTo=bTu2a1K-F19w), we'll use DocParse to extract data from an NTSB report. We'll go through the important code snippets below to see what's going on. For the purposes of this tutorial, we focus on page 2 of the document, given below.
+
+<Frame>
+  <img src="../images/59_page2.png" />
+</Frame>
+
+## Getting Parsed Document
+
+### Making a Call to DocParse
+
+```python text_demo.py
+with open(file_name, 'rb') as file:
+  ## Make a call to the partitioning service and set extract_images to true.
+  partitioned_file = partition_file(file, aryn_api_key, extract_images=True, extract_table_structure=True, use_ocr=False, selected_pages=[2])
+```
+We use the `partition_file` function from `aryn_sdk.partition` to extract text and images from the document. The `aryn_api_key` is your API key from Aryn. Set `extract_images` and `extract_table_structure` to `True` to extract images and tables. Set `use_ocr` to `False` to avoid OCR and use embedded text. We use `selected_pages=[2]` to focus on page 2.
+
+### Viewing the JSON Output
+
+```json output.json
+{'status': ['Incremental status will be shown here during execution.',
+  "Until you get a line that matches '  ]\n', you can convert the partial",
+  'output to a json document by appending \'""]}\' to the partial output.',
+  '',
+  'T+   0.00: Server version managed-service-22-model-1-4 Model version 1.4',
+  'T+   0.00: Received request with aryn_call_id=REDACTED',
+  'T+   0.00: Waiting for scheduling',
+  'T+   0.01: Preprocessing document',
+  'T+   0.01: Done preprocessing document',
+  'T+   0.51: Completed work on page 2',
+  ''],
+ 'elements': [{'type': 'Image',
+   'bbox': [0.0873676793715533,
+    0.08911955399946733,
+    0.8548237161075367,
+    0.5357984508167614],
+   'properties': {'score': 0.873895525932312,
+    'image_size': [1324, 1003],
+    'image_mode': 'RGB',
+    'image_format': None,
+    '_element_index': 1,
```

Comment:
is this 1 indexed or 0 indexed? Should we talk a bit about that?

Reply:
This shouldn't be in the output, have removed it.